### PR TITLE
V4.02 CSD01 release candidate

### PR DESCRIPTION
**Pull request by ralfhandl**
- [x] Updated dates in meta.yaml files
- [x] Rebuilt MD, HTML, and PDF
- [x] Packaged ZIP files

### CODE REVIEW

**Review by GEricson — DISMISSED** [review]

**Review by HeikoTheissen — APPROVED** [review]

**Review by HeikoTheissen — CHANGES_REQUESTED** [review]
[OData-CSDL, section 1.1](https://github.com/oasis-tcs/odata-specs/blob/bc64c2f30853c9421c5344f2347178970692b959/docs/odata-csdl-json/odata-csdl-json.md#11-changes-from-earlier-versions) should be sorted by sections.

File: lib/build-pdf.js [hunk 1]
```diff
@@ -4,8 +4,8 @@ const pdf = require("./pdf.js");
 const doc = process.argv[2];
 
 iterator(function (srcname, name, variant, meta) {
-  if (doc && doc !== name) return;
-  pdf(name, variant, meta, doc === name)
+  if (doc !== "*" && doc !== name) return;
```
Comment (ralfhandl) [inline lib/build-pdf.js:7]: This change means that PDFs are never generated

**Comment by HeikoTheissen** [comment]
Must we include ABNF or vocabulary files in the ZIP folders?

**Review by HeikoTheissen — APPROVED** [review]

**Comment by ralfhandl** [comment]
> Must we include ABNF or vocabulary files in the ZIP folders?

If they are listed in "Additional artifacts" on the title page, we have to include them in the ZIP file.

**Review by GeraldKrause — APPROVED** [review]

**Comment by HeikoTheissen** [comment]
> If they are listed in "Additional artifacts" on the title page, we have to include them in the ZIP file.

They are not, although the text sometimes refers to certain vocabulary terms.

And changes we made to the context URL and to the URL conventions are also reflected in the ABNF. Doesn't that make the ABNF an additional artifact for them?

**Comment by ralfhandl** [comment]
Protocol and URL Conventions are bundled together and have the ABNF and testcases as additional artifacts.

CSDL JSON has the JSON schema file as additional artifact, CSDL XML the two XML schema file.

JSON Format has no additional artifacts.

**Review by HeikoTheissen — DISMISSED** [review]

**Review by HeikoTheissen — APPROVED** [review]

**Review by HeikoTheissen — APPROVED** [review]

**Pull request merged**